public/javascript/APIHandler.js

```diff
@@ -1,25 +1,21 @@
+
 class APIHandler {
   constructor (baseUrl) {
     this.BASE_URL = baseUrl;
+    this.api = axios.create({
+        baseURL: baseUrl
+    })
   }
 
-  getFullList () {
-
-  }
-
-  getOneRegister () {
-
-  }
-
-  createOneRegister () {
-
-  }
+  getFullList = () => this.api.get('/characters')
 
-  updateOneRegister () {
+  getOneRegister = id => this.api.get(`characters/${id}`)
 
-  }
+  createOneRegister = character => this.api.get('/characters', character)
```

Comment:
Cuidado, este método es igual que el primero, la url para crear era la misma pero usando POST.

```suggestion
  createOneRegister = character => this.api.post('/characters', character)
```